docs/developer/Technical Documentation/Dev Process/Hpw to contribute.md

```diff
@@ -0,0 +1,121 @@
+# Contribution details
```

Comment:
typo in file name

Reply:
fixed with: https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/pull/1/commits/88c2441f820cfb0be2bb85246e33ae7235090154